tests/h/search/client_test.py

```diff
@@ -5,33 +5,29 @@
 
 
 class TestClient:
-    def test_it_sets_the_index_property(self):
```

Comment:
These tests were previously able to get away with not disposing of the `Client` properly. That is no longer the case because the constructor now makes a call to the server to get its version, establishing a connection which needs to be closed using `client.close()`.